app/src/main/java/com/framgia/soundclound/screen/main/MainActivity.java

```diff
@@ -0,0 +1,52 @@
+package com.framgia.soundclound.screen.main;
+
+import android.databinding.DataBindingUtil;
+import android.graphics.PorterDuff;
+import android.os.Bundle;
+import android.support.design.widget.TabLayout;
+import android.support.v4.content.ContextCompat;
+import android.support.v4.view.ViewPager;
```

Comment:
ERROR: Unused import: android.support.v4.view.ViewPager. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports